tests/test_data_types.py

```diff
@@ -161,13 +157,16 @@ def test_array_to_dict():
     description = 'A description about this variable.'
     arr = array(item_type=text, description=description, min_length=5, max_length=10)
     obj = arr.to_dict()
-    assert obj['name'] == 'text_array'
+
     assert obj['type'] == 'array'
-    assert obj['itemType'] == text().to_dict()
     assert obj['minLength'] == 5
     assert obj['maxLength'] == 10
     assert obj['description'] == description
 
+    tmp = text()
+    tmp.name = 'text_array_item'
```

Comment:
Maybe we should just allow data types that are in arrays to have the name `None` instead of `"%s_array_item" % type`...?